mpf/tests/machine_files/mode_tests/modes/mode_in_sub_folder/sub_mode/mode6/config/mode6.yaml

```diff
@@ -0,0 +1,14 @@
+#config_version=5
+mode:
+  start_events: start_mode1
```

Comment:
Small nitpick: This should be start_mode6. Otherwise other tests will accidentally start this mode. Same for the other new modes